mimic/rest/cinder_api.py

```diff
@@ -31,7 +31,14 @@ def catalog_entries(self, tenant_id):
         """
         return [
             Entry(
-                tenant_id, "volume", "cloudBlockStorage",
```

Comment:
Devstack Mitaka uses the name `cinder`
 * https://git.openstack.org/cgit/openstack-dev/devstack/tree/lib/cinder?h=stable%2Fmitaka#n364